### ISSUE TITLE
[jupyter-kernel] Exceptions do not appear in `jupyter console --kernel deno`

### ISSUE THREAD

**Issue by mmastrac** [denoland/deno#20553]
This is _mostly_ fixed by #20531 but I am still seeing it locally in a jupyter console (v6.6.3):

```
10:25 $ jupyter console --kernel deno
Jupyter console 6.6.3

Welcome to Deno kernel
In [1]: throw "hi"

In [2]:                                                             
```

**Comment by rgbkrk** [comment]
If I had to guess, the console is expecting to get the `traceback` field and we're just not sending it on. I started on that in https://github.com/denoland/deno/pull/20530 but was feeling way out of my depths for getting the precise error format needed.

**Comment by mmastrac** [comment]
@rgbkrk Ahh. Yes, I just confirmed that jupyter console is wanting something to appear the traceback.

**Comment by mmastrac** [comment]
Fixed by #20530
